src/app/docs/datepicker/datepicker-docs.component.html

```diff
@@ -7,14 +7,107 @@
     The datepicker module provides an input and calendar for users to select dates or fuzzy dates.
   </sky-docs-demo-page-summary>
 
-  <!-- <sky-docs-code-examples
+  <sky-docs-demo
```

Comment:
Do we really need a separate header and border for each demo? Or could they be combined in the same box with a `Demo` header like how we have the standard checkboxes and radio buttons alongside the icon checkboxes and radio buttons? Not sure if it matters, but just throwing it out there.

Reply:
I'm just going off the design spec: https://blackbaud.invisionapp.com/share/T7RPR60C5NG#/screens/360590459_datepicker

Reply:
@Blackbaud-AdamFunderburk, any opinion on this? I assume we're good to push this PR through as is, but just curious if there is a rationale behind when we have multiple demos at the top of the page and when we have multiple items within a single demo. (In this case, I might not even have registered that they were separate except that when I was looking for an ARIA label in the code sample, I realized we don't have an ARIA label for this component and wondered if in lieu of an ARIA label, a regular label would be at least a slight nod toward acccessibility.)